packages/builders/src/jest/schema.json

```diff
@@ -62,6 +62,10 @@
       "type": "string",
       "alias": "t",
       "description": "Run only tests with a name that matches the regex. (https://jestjs.io/docs/en/cli.html#testnamepattern-regex)"
+    },
+    "reporters": {
+      "type": "string|json",
```

Comment:
This syntax is invalid. Also, I think it should be `string` or `Array<string>`.

You can use `oneOf`:

Example: https://github.com/nrwl/nx/blob/master/packages/builders/src/web/build/schema.json#L41